apps/molgenis-components/src/components/tables/TableExplorer.vue

```diff
@@ -624,6 +631,16 @@ export default {
     this.setTableMetadata(newTableMetadata);
     await this.reload();
   },
```

Comment:
RoutedTableExplorer limit setting breaks it now stores ([object+Event] in the url)

https://preview-emx2-pr-1778.dev.molgenis.org/apps/molgenis-components/#/component/RoutedTableExplorer?_limit=[object+Event]

https://emx2.dev.molgenis.org/apps/molgenis-components/#/component/RoutedTableExplorer?_limit=10
here it stores the limit value

Reply:
indeed a bug. Missed an @input that should become @update:modelValue.